.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+    release:
+        types: [published]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Extract tag version number
+              id: get_version
+              uses: battila7/get-version-action@v2
```

Comment:
maybe we could use an alternative here. It seems like that particular action is not maintained anymore and uses some deprecated code (battila7/get-version-action#22)

Reply:
seems like we could just use `${GITHUB_REF_NAME#v}` and omit this step entirely

Reply:
I just removed this step and replaced it with what you suggested. Also replaced `npm ci` with `yarn install` since it's what's used in the `build.sh` script

Reply:
lgtm!